### PR TITLE
Poweredge r750xs

### PR DESCRIPTION
**Pull request by max0013**

### CODE REVIEW

**Comment by danner26** [comment]
Why do you have two different models, one with esxi? The purpose of this library is for the physical devices themselves. Please ensure you do not have redundant devices in a PR

**Comment by max0013** [comment]
oh yeah, it was for testing something, its presence in the commit is a mistake sorry. 
I just delete it

File: device-types/Dell/PowerEdge-R750xs.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+manufacturer: Dell
+model: PowerEdge R750xs
+slug: dell-poweredge-r750xs
+u_height: 2
+is_full_depth: true
+front_image: true
+rear_image: true
```
Comment (danner26) [inline device-types/Dell/PowerEdge-R750xs.yml:8]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/Dell/PowerEdge-R750xs.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+comments: |
+  [PowerEdge R750xs](https://dl.dell.com/content/manual49765834-dell-emc-poweredge-r750xs-manuel-d-installation-et-de-maintenance.pdf?language=fr-fr)
+rear-ports:
+  - name: Rear VGA
+    type: other
+  - name: Rear USB 2
+    type: other
+  - name: Rear USB 3
+    type: other
```
Comment (danner26) [inline device-types/Dell/PowerEdge-R750xs.yml:17]: Video and USBs are not suppose to be modelled here. Please remove these

File: device-types/Dell/PowerEdge-R750xs.yml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  - name: iDrac
+    type: other
```
Comment (danner26) [inline device-types/Dell/PowerEdge-R750xs.yml:19]: If this refers to the iDRAC serial port, please model this under `console-ports` with the proper interface type

File: device-types/Dell/PowerEdge-R750xs.yml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+front-ports:
+  - name: Front VGA
+    type: other
+    rear_port: Rear VGA
+  - name: Front USB 2
+    type: other
+    rear_port: Rear USB 2
```
Comment (danner26) [inline device-types/Dell/PowerEdge-R750xs.yml:26]: Video and USBs are not suppose to be modelled in NetBox at the moment. Please remove them

File: device-types/Dell/PowerEdge-R750xs.yml [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  - name: Front USB micro-AB
+    type: other
+    rear_port: iDrac
```
Comment (danner26) [inline device-types/Dell/PowerEdge-R750xs.yml:29]: This should be modelled under the `console-ports` section with the proper interface type

File: device-types/Dell/PowerEdge-R750xs.yml [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+power-ports:
+  - name: Power 1
+    type: iec-60320-c14
+  - name: Power 2
+    type: iec-60320-c14
```
Comment (danner26) [inline device-types/Dell/PowerEdge-R750xs.yml:34]: These PSUs are modular. You should be modelling this as `module-bays`, and the associated PSU modules should be created in the module-types folder if they do not already exist

File: device-types/Dell/PowerEdge-R750xs.yml [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+    mgmt_only: true
+module-bays:
+  - name: AHCI controller in SL 10
+    position: AHCI.SL.10-1
+    label: BOSS-S2
+  - name: Backplane 1 on Connector 0 of RAID Controller in Slot 1
+    position: Internal.0-1:RAID.Slot.1-1
+    label: BP_PSV 0:1
+  - name: Battery on RAID Controller in Slot 1
+    position: Integrated.1:RAID.Slot.1-1
```
Comment (danner26) [inline device-types/Dell/PowerEdge-R750xs.yml:51]: Currently storage is not suppose to be modelled in NetBox. We have an open FR to work on that, but it is not currently ready. Please remove these definitions.

File: device-types/Dell/PowerEdge-R750xs.yml [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+    label: DDR4 DIMM
+  - name: DIMM B2
+    position: DIMM.Socket.B2
+    label: DDR4 DIMM
+  - name: DIMM B3
+    position: DIMM.Socket.B3
+    label: DDR4 DIMM
+  - name: DIMM B4
+    position: DIMM.Socket.B4
+    label: DDR4 DIMM
```
Comment (danner26) [inline device-types/Dell/PowerEdge-R750xs.yml:81]: Unfortunately these should not be modelled in NetBox at this time. Please remove them.

**Review by danner26 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**